components/atomic/Loader/styles.js

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+import { CircularLoader } from "@rubin-epo/epo-react-lib";
+
+export const Loader = styled(CircularLoader)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Styles just center it horizontally and vertically in its parent.

Reply:
do you think this change should go upstream to the library?

Reply:
Probably?  Feels like a pretty sensible default styling for the spinner.  And wouldn't be much to override if ever wanted to use otherwise.